packages/grid/x-data-grid-premium/src/typeOverloads/index.ts

```diff
@@ -1 +1 @@
-export * from './modules';
+import '@mui/x-data-grid-pro/typeOverloads';
```

Comment:
Just to be clear we went with duplicating the row grouping required files over to the `x-data-grid-pro` package, right? I'm asking because I don't see them being removed from the `x-data-grid-premium`. That's ok, but can we add somewhere, maybe a TODO explaining what needs to happen (i guess as simple as reverting this exact commit) once the transition period is over?

Reply:
Yes I went for duplication to avoid spending to much time on a temporary solution
I am open to discussion of course.